plots/measure.js

```diff
@@ -43,13 +44,25 @@ const args = require('yargs')
 
 const constants = require('./constants.js');
 const utils = require('./utils.js');
-const config = require('../lighthouse-core/config/plots.json');
+const config = require('../lighthouse-core/config/plots-config.js');
 const lighthouse = require('../lighthouse-core/index.js');
 const ChromeLauncher = require('../chrome-launcher/chrome-launcher.js');
 const Printer = require('../lighthouse-cli/printer');
 const assetSaver = require('../lighthouse-core/lib/asset-saver.js');
 
 const keepFirstRun = args.keepFirstRun || !args.reuseChrome;
+const outPath = generateOutPath();
+
+/**
+ * @return {string}
+ */
+function generateOutPath() {
+  if (args.out) {
+    return path.resolve(__dirname, args.out);
+  }
+  const date = new Date().toISOString();
```

Comment:
you'll need to replace colons in the ISOString

![image](https://user-images.githubusercontent.com/39191/27112699-80e69476-506d-11e7-84fa-62f9418e9c3f.png)